### ISSUE TITLE
warning: Could not read SDKSettings.json for SDK

### ISSUE THREAD

**Issue by toffaletti** [swiftlang/swift-sdk-generator#110]
custom SDKs generated with `swift-sdk-generator` cause `swift build` to repeatedly output this warning:

```
warning: Could not read SDKSettings.json for SDK
```

This can be seen in logs from other issues:
https://github.com/apple/swift-sdk-generator/issues/12
https://github.com/apple/swift-sdk-generator/issues/27

**Comment by xtremekforever** [comment]
@MaxDesiatov @toffaletti I did a little research today about this issue and found that this `SDKSettings.json` file is used by the [SwiftCrossCompilers](https://github.com/CSCIX65G/SwiftCrossCompilers) project. Check out the reference file here: https://github.com/CSCIX65G/SwiftCrossCompilers/blob/master/SDKSettings.json

I tried a very minimal file to see what would work to just silence the warning. Including the following content in `SDKSettings.json` in the SDK directory (the ubuntu-jammy.sdk directory, for example) works:

```json
{
  "SupportedTargets": {},
  "Version": "0.0.1",
  "CanonicalName": "linux"
}
```

Is it worth including this file by default in the generated SDKs to silence this warning and make the output cleaner? It seems like this is a feature of swiftpm now and unless it's removed, it's going to search for the file whenever cross-compiling is happening. 

In my SDK I'm adding this empty file to make the compilation output cleaner. It doesn't hurt anything although it's annoying that it has to be there just to make the warning go away.
